ui/plugins/eu.esdihumboldt.hale.ui.views.report/src/eu/esdihumboldt/hale/ui/views/report/ReportListLabelProvider.java

```diff
@@ -93,7 +93,11 @@ public String getText(Object element) {
 			return ((Report<?>) element).getTaskName();
 		}
 		else if (element instanceof ReportSession) {
-			return df.format(new Date(((ReportSession) element).getId()));
+			long id = ((ReportSession) element).getId();
+			if (id == 0) {
```

Comment:
We could use a constant for the unknown/undetermined id here instead of a magic number